src/forms/_base.scss

```diff
@@ -226,6 +235,11 @@ select {
     transform: translateY(1rem) scale(1);
     transition: transform 0.25s ease-out;
     transform-origin: 0 0;
+    text-transform: none;
+
+    + .form-input {
```

Comment:
I'd actually prefer to remove the `&` on line 195 as it's not actually necessary. I prefer only using the `&` when you need to change the default nesting order, as in `.foo &`. Thoughts?